_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -206,6 +206,95 @@ You will most likely not need to specify any parameters except for `location`. F
 
 You will most likely not need to specify any parameters except for `bucket` and `base_path`. For allowed request parameters, see [Register or update snapshot repository API](https://opensearch.org/docs/latest/api-reference/snapshots/create-repository/).
 
+
+### Azure storage account
+
+These steps explain how to register a snapshot repository backed by an Azure storage account for an OpenSearch cluster deployed using Helm.
+
+1. Create an Azure storage account. Then create a container within the storage account.
+
+1. Create a Docker image with the `repository-azure` plugin.
+
+   create-keystore.yaml
+
+   ```bash
+   #!/bin/bash
+
+   /usr/share/opensearch/bin/opensearch-keystore create
+   echo $AZURE_SNAPSHOT_STORAGE_ACCOUNT | /usr/share/opensearch/bin/opensearch-keystore add --stdin azure.client.default.account
+   echo $AZURE_SNAPSHOT_STORAGE_ACCOUNT_KEY | /usr/share/opensearch/bin/opensearch-keystore add --stdin azure.client.default.key
+   cp /usr/share/opensearch/config/opensearch.keystore /tmp/keystore/opensearch.keystore
+   ```
+
+   Dockerfile
```

Comment:
```suggestion
1. Create a Docker image with the `repository-azure` plugin using the following commands:
```

Reply:
Since this section has both the Dockerfile and the command to build it and also creates the keystore, shall we say `Create a Docker image with the repository-azure plugin and keystore using the following Dockerfile and command:`

Reply:
Let's separate these into two different steps. I'll make the adjustments on the file directly and let you know once I'm done.

Reply:
Done.